src/60_Ads_for_AMP_(A4A)/A4A_Carousel.html

```diff
@@ -39,35 +40,29 @@
   <amp-carousel width="300"
                 height="250"
                 type="slides">
-    <div class="slide">
-      <a target="_blank"
-         href="<%host%>/ads_for_amp_a4a/a4a_carousel/">
-        <amp-img src="/img/a4a_slide1.png"
-                 width=300
-                 height=250
-                 alt="slide 1"/>
-      </a>
-    </div>
+    <a target="_blank"
+       href="<%host%>/ads_for_amp_a4a/a4a_carousel/">
+       <amp-img src="<%host%>/img/a4a_slide1.png"
```

Comment:
@carlosfuertes FYI I had to make all image paths absolute as the ads are served in the preview from a different domain. Is that OK from you side or does it send the wrong signal?